doc/release-notes.md

```diff
@@ -47,6 +47,9 @@ Notable changes
 P2P and network changes
```

Comment:
Another idea would be to also remove this file completely (There is `doc/release-notes-empty-template.md`) to clarify that release notes are created in the wiki either way and not here in the master branch?

Reply:
> Another idea would be to also remove this file completely 

Do you mean remove it as part of this pull, and at the same time, move the content over to the wiki?

Reply:
Yeah, if you agree that it can be removed, then it can be done as part of this pull.


Removing it means that snippets will only be merged in the wiki and not in the master branch.

Reply:
Ok. I've created a page on the wiki here: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/24.0-Release-Notes-draft, from the consolidated contents of release-notes.md. I'll push again to drop release-notes.md.